package.json

```diff
@@ -43,5 +43,8 @@
     "*.{json,md}": [
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "react-query": "^3.39.3"
   }
```

Comment:
This dependency is at the wrong package.json file.

You should add it on the `apps/web` page